components/camel-mail/src/generated/resources/META-INF/javamail.default.providers

```diff
@@ -1,9 +1,9 @@
 # Jakarta Mail IMAP provider Oracle
-protocol=imap; type=store; class=com.sun.mail.imap.IMAPStore; vendor=Oracle;
-protocol=imaps; type=store; class=com.sun.mail.imap.IMAPSSLStore; vendor=Oracle;
+protocol=imap; type=store; class=org.eclipse.angus.mail.imap.IMAPStore; vendor=Oracle;
```

Comment:
I assume maybe vendor should be changed to Eclipse?

Reply:
Thanks, let me change it.

Reply:
It is copied verbatim from angus-mail, so there is not much I can do with it: https://github.com/eclipse-ee4j/angus-mail/blob/5622551fc384dc87b1cdd15bd309b3d4cc1f137d/providers/angus-mail/src/main/resources/META-INF/javamail.default.providers#L2